Makefile.am

```diff
@@ -230,9 +230,9 @@ rpmvar_DATA =
 
 install-exec-hook:
 	@rm -f $(DESTDIR)$(bindir)/rpmquery
-	@LN_S@ ../../bin/rpm $(DESTDIR)$(bindir)/rpmquery
+	@LN_S@ rpm $(DESTDIR)$(bindir)/rpmquery
```

Comment:
I would suggest making a new pull request that adds a configure switch for changing the behavior, with the default behavior being set to the current one, while offering the ability to switch to the alternative one (where it follows the prefix'd path).